datacube_utilities/query.py

```diff
@@ -26,8 +27,9 @@ def create_base_query(aoi, res, output_projection, aoi_crs, dask_chunks):
         "output_crs": output_projection,
         "resolution": resolution,
         "dask_chunks": dask_chunks,
-        "crs": aoi_crs,
+        "crs": "EPSG:3460",
```

Comment:
I would avoid to hardcode the crs

Reply:
I would like to avoid this too. Let me have a think. Something with default parameters will probably work